test_conformance/extensions/cl_khr_command_buffer/negative_tests/command_buffer_create.cpp

```diff
@@ -0,0 +1,318 @@
+//
+// Copyright (c) 2024 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "../basic_command_buffer.h"
+#include "procs.h"
+
+
+//--------------------------------------------------------------------------
+namespace {
+
+// CL_INVALID_VALUE if num_queues is not one.
```

Comment:
Can we add a `Skip()` method here that checks `cl_khr_command_buffer_multi_device` is not a supported device extension. See this error language https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_Ext.html#_error_updates_2 of how the extension changes the error "CL_INVALID_VALUE if num_queues is not one." to "CL_INVALID_VALUE if num_queues is zero."